utils/rpms/cart.spec

```diff
@@ -137,6 +139,12 @@ ln %{?buildroot}%{carthome}/{TESTING/.build_vars,.build_vars-Linux}.sh
 %{carthome}/.build_vars-Linux.sh
 
 %changelog
+* Mon Nov 11 2019 Brian J. Murrell <brian.murrell@intel.com> - 1.6.0-4
+- Don't R: ompi-devel from cart-devel as it breaks the ior
+  build whic ends up building with ompi instead of mpich
```

Comment:
typo whic

Reply:
In the interest of expediency of getting hardware testing going again, I will be happy to fix this in a following iteration if that's acceptable.